Networking/WebSockets/WebSocketImpl.cc

```diff
@@ -127,13 +128,12 @@ namespace litecore { namespace websocket {
     }
 
     void WebSocketImpl::onConnect() {
-        if(_closed) {
-            // If the WebSocket has already been closed, which only happens in rare cases
-            // such as stopping a Replicator during the connecting phase, then don't continue...
-            warn("WebSocket already closed, ignoring onConnect...");
+        int expected = SOCKET_OPENING;
+        if (! atomic_compare_exchange_strong(&_socketLCState, &expected, (int)SOCKET_OPENED) ){
+            logInfo("WebSocket already closed or is closing, ignoring onConnect...");
```

Comment:
This message doesn't seem to be correct anymore in every case.  If the state is already `OPENED` then the web socket is not "already closed or is closing".  Perhaps the message should be altered:

"WebSocket not in 'opening' state, ignoring onConnect..."